tests/ImageSharp.Tests/Formats/Jpg/JpegColorConverterTests.cs

```diff
@@ -433,29 +436,29 @@ static JpegColorConverter.ComponentValues Copy(JpegColorConverter.ComponentValue
                 case JpegColorSpace.YCbCr:
                     ValidateYCbCr(original, result, i);
                     break;
+                case JpegColorSpace.Undefined:
                 default:
-                    Assert.True(false, $"Colorspace {colorSpace} not supported!");
+                    Assert.True(false, $"Invalid Colorspace enum value: {colorSpace}.");
                     break;
             }
         }
 
-        private static void ValidateYCbCr(in JpegColorConverter.ComponentValues values, in JpegColorConverter.ComponentValues result, int i)
+        private static void ValidateYCbCr(in JpegColorConverterBase.ComponentValues values, in JpegColorConverterBase.ComponentValues result, int i)
         {
             float y = values.Component0[i];
             float cb = values.Component1[i];
             float cr = values.Component2[i];
-            var ycbcr = new YCbCr(y, cb, cr);
+            var expected = ColorSpaceConverter.ToRgb(new YCbCr(y, cb, cr));
 
             var actual = new Rgb(result.Component0[i], result.Component1[i], result.Component2[i]);
-            var expected = ColorSpaceConverter.ToRgb(ycbcr);
 
-            Assert.Equal(expected, actual, ColorSpaceComparer);
+            bool equal = ColorSpaceComparer.Equals(expected, actual);
+            Assert.True(equal, $"Colors {expected} and {actual} are not equal at index {i}");
         }
 
-        private static void ValidateCyyK(in JpegColorConverter.ComponentValues values, in JpegColorConverter.ComponentValues result, int i)
+        private static void ValidateCyyK(in JpegColorConverterBase.ComponentValues values, in JpegColorConverterBase.ComponentValues result, int i)
         {
-            var v = new Vector4(0, 0, 0, 1F);
-            var scale = new Vector4(1 / 255F, 1 / 255F, 1 / 255F, 1F);
+            Vector4 v = Vector4.Zero;
```

Comment:
@br3aker could you try replacing this Vector4.Zero with the explicit `Vector4(0, 0, 0, 1F)`? I had a similar strange issue in the past when using Vector default values with `net472` and `netcoreapp2.0`. See: https://github.com/dotnet/runtime/issues/55623

Reply:
I remember that issue but unfortunately all tests with Vector code fail - even YCbCr which does not use Vector4 at all for comparison.

Reply:
I bet its some similar strange issue.